relay-dynamic-config/src/defaults.rs

```diff
@@ -395,5 +395,73 @@ fn span_metrics(is_extract_all: bool) -> impl IntoIterator<Item = MetricSpec> {
                     .always(), // already guarded by condition on metric
             ],
         },
+        MetricSpec {
+            category: DataCategory::Span,
+            mri: "c:spans/dynamically_loaded_libraries_count@none".into(),
+            field: Some("span.measurements.dynamically_loaded_libraries_count".into()),
+            condition: Some(
+                duration_condition.clone() & is_mobile.clone() & app_start_condition.clone(),
+            ),
+            tags: vec![
+                Tag::with_key("span.op")
+                    .from_field("span.sentry_tags.op")
+                    .always(), // already guarded by condition on metric
+                Tag::with_key("span.description")
+                    .from_field("span.sentry_tags.description")
+                    .always(), // already guarded by condition on metric
+                Tag::with_key("span.group")
+                    .from_field("span.sentry_tags.group")
+                    .always(), // already guarded by condition on metric
+                Tag::with_key("transaction")
+                    .from_field("span.sentry_tags.transaction")
+                    .always(), // already guarded by condition on metric
+                Tag::with_key("device.class")
+                    .from_field("span.sentry_tags.device.class")
+                    .always(), // already guarded by condition on metric
+                Tag::with_key("release")
+                    .from_field("span.sentry_tags.release")
+                    .always(), // already guarded by condition on metric
+                Tag::with_key("os.name")
+                    .from_field("span.sentry_tags.os.name")
+                    .always(), // already guarded by condition on metric
+                Tag::with_key("environment")
+                    .from_field("span.sentry_tags.environment")
+                    .always(), // already guarded by condition on metric
```

Comment:
If we're unsure what tags we're going to need, let's start with the bare minimum and only add tags later if we really need them. This will result in fewer rows in clickhouse, i.e. faster queries & faster UX.